src/test/java/io/nats/client/JetstreamTests.java

```diff
@@ -88,69 +79,52 @@ public void testStreamAndConsumerCreate() throws IOException, InterruptedExcepti
     }
 
     @Test
-    public void testJetstreamPublishDefaultOptions() throws IOException, InterruptedException, ExecutionException {
+    public void testJetstreamPublishDefaultOptions() throws IOException, InterruptedException, TimeoutException {
         try (NatsTestServer ts = new NatsTestServer(false, true); Connection nc = Nats.connect(ts.getURI())) {
-
-            try {
-                JetStream js = nc.jetStream();
-                createMemoryStream(js, "foo-stream", "foo");
-
-                PublishAck ack = js.publish("foo", null);
-                assertEquals(1, ack.getSeqno());
-            } catch (Exception ex) {
-                Assertions.fail("Exception:  " + ex.getMessage());
-            } finally {
-                nc.close();
-            }
```

Comment:
1. If an exception is throw the test will fail naturally, so no need to check for it.
2. Finally to close the connection is not required since Connection implements AutoCloseable and the object is created via the try, the close happens automatically.